gazebo/rendering/Camera.hh

```diff
@@ -911,6 +941,9 @@ namespace gazebo
       /// \brief Flag for signaling the usage of camera intrinsics within OGRE
       protected: bool cameraUsingIntrinsics;
 
+      /// \brief Camera Intrinsic Matrix
+      protected: ignition::math::Matrix3d cameraIntrinsicMatrix;
```

Comment:
FYI this breaks ABI. If it can be private instead of `protected`, you can put it in https://github.com/osrf/gazebo/blob/gazebo11/gazebo/rendering/CameraPrivate.hh